### ISSUE TITLE
accounts/graphql-client bundle size

### ISSUE THREAD

**Issue by sakulstra** [accounts-js/accounts#938]
# Bug report
## Describe the bug
When adding @accounts/graphql-client we realized that our bundle size increased by nearly 150kb - it turns out to be related to how `graphql` is bundled within the package - not quite sure yet how to fix it though.

## To Reproduce
Add the dependency to your project.

## Expected behavior
not having the complete graphql package in the build/ having a smaller footprint when using this module
- Version of accounts-js: [e.g. 0.0.24]

### Additional info
So it seems like this is not only a problem with specifically the "graphql" package and most likely an issue with all the client side packages in here. The "problem" from my understanding seems to be that using tsc to compile stuff: https://github.com/accounts-js/accounts/blob/master/packages/graphql-client/package.json#L11 doesn't do fancy things like tree shaking - this leads to complete node/modules ending up in the bundle when e.g. importing `{sth} from 'graphql'`.

**Comment by nsine** [comment]
tldr;
bundle size is super huge because of https://github.com/accounts-js/accounts/blob/master/packages/graphql-client/src/graphql-client.ts#L9
will create a pr to fix this

**Comment by pradel** [comment]
Fixed in v0.25.1
